src/patchelf.cc

```diff
@@ -1855,12 +1876,12 @@ void ElfFile<ElfFileParamNames>::addDebugTag()
     newDyn.d_un.d_val = 0;
     setSubstr(newDynamic, 0, std::string((char *) &newDyn, sizeof(Elf_Dyn)));
 
-    this->rewriteSections();
+    rewriteSections();
     changed = true;
 }
 
 template<ElfFileParams>
-void ElfFile<ElfFileParamNames>::clearSymbolVersions(const std::set<std::string> & syms)
+void ElfFile<ElfFileParamNames()>::clearSymbolVersions(const std::set<std::string> & syms)
```

Comment:
It would be nicer if these changes weren't made in this PR, or at least in a different commit

Reply:
I'll (try to) split it in cosmetic commit and functional commit.